src/main/scala/cromwell/engine/ExecutionStore.scala

```diff
@@ -20,14 +20,19 @@ class ExecutionStore(namespace: WdlNamespace) {
   var table = namespace.workflows.head.calls.map { call => call -> ExecutionStatus.NotStarted }.toMap
 
   /**
-   * Return all calls which are currently in state `NotStarted` and whose prerequisites are all `Done`,
+   * Start all calls which are currently in state `NotStarted` and whose prerequisites are all `Done`,
    * i.e. the calls which should now be eligible to run.
    */
-  def runnableCalls: Iterable[Call] = {
-    for {
+  def startRunnableCalls: Iterable[Call] = {
+    val runnableCalls = for {
       callEntry <- table if callEntry._2 == ExecutionStatus.NotStarted
       call = callEntry._1 if callIsRunnable(call)
     } yield call
+
+    runnableCalls foreach { call =>
+      table += call -> ExecutionStatus.Starting
```

Comment:
`table` is a `Map[Call, ExecutionStatus]` which is known to already contain an entry for this particular `Call`, so this is a replacement.